airgun/views/cloud_inventory.py

```diff
@@ -113,6 +114,8 @@ class CloudInventoryListView(BaseLoggedInView):
     )
     obfuscate_ips = Switch('.//label[@for="rh-cloud-switcher-obfuscate_inventory_ips"]')
     exclude_packages = Switch('.//label[@for="rh-cloud-switcher-exclude_installed_packages"]')
+    cloud_connector = Pf4Button(locator='//button[text()="Configure Cloud Connector"]')
+    sync_status = Pf4Button(locator='//button[text()=" Sync inventory status"]')
```

Comment:
if possible use the contains in the locator rather having adding ` ` extra space

Reply:
Even better, you should be able to use the OUIA component ID only, and not write an xpath locator at all when using PF4 OUIA compatible widgets.

Please import PF4 widgets through `widgetastic_patternfly4.ouia`